packages/app-store/zohocalendar/_metadata.ts

```diff
@@ -0,0 +1,27 @@
+import type { AppMeta } from "@calcom/types/App";
+
+import _package from "./package.json";
+
+export const metadata = {
+  name: "Zoho Calendar",
+  description: _package.description,
+  installed: true,
+  type: "zoho_calendar",
+  title: "Zoho Calendar",
+  imageSrc: "/api/app-store/zohocalendar/icon.svg",
```

Comment:
```suggestion
  imageSrc: "/api/app-store/zohocalendar/icon.svg",
```
It's deprecated and not needed in any new app. `logo` does the same job